ConsoleApp/Stack.cs

```diff
@@ -0,0 +1,69 @@
+namespace ConsoleApp;
+class LinkedStack<T>
+{
+    private record StackItem
+    {
+        public T Value { get; init; }
+        public StackItem Next { get; set; }
+    }
+
+    private StackItem _head;
+    public int Length { get; private set; }
```

Comment:
it should name differently
```suggestion
    public int Count { get; private set; }
```